x/crosschain/types/message_add_inbound_tracker.go

```diff
@@ -48,17 +47,7 @@ func (msg *MsgAddInboundTracker) ValidateBasic() error {
 	if err != nil {
 		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
-	chain := chains.GetChainFromChainID(msg.ChainId)
-	if chain == nil {
-		return errorsmod.Wrapf(ErrInvalidChainID, "chain id (%d)", msg.ChainId)
-	}
-	if msg.Proof != nil && !chain.SupportMerkleProof() {
-		return errorsmod.Wrapf(
-			ErrProofVerificationFail,
-			"chain id %d does not support proof-based trackers",
-			msg.ChainId,
-		)
-	}
+
```

Comment:
The method `SupportMerkleProof` is used no where. Should we remove?

Reply:
Replacing `ErrProofVerificationFail` with `sdkerrors.ErrInvalidRequest` there might have been a merge issue